classes/03class/exercises/c03-compose01/jayanath/docker-compose.yaml

```diff
@@ -0,0 +1,21 @@
+version: "3.8"
+services:
+  db:
+    image: mysql:5.7
+    environment:
+      MYSQL_ROOT_PASSWORD: fakeroot
+
+  web:
+    build:
+      context: ../../../compose/artifacts/c05-compose01/
+    environment:
+      DB_USER: root
+      DB_PASS: fakeroot
+      DB_HOST: db
```

Comment:
Good usage of the internal Docker network, using `db` as the host without exposing ports